packages/mdc-card/README.md

```diff
@@ -158,6 +158,8 @@ You can use the `mdc-card__actions--vertical` option to lay actions out vertical
 Be sure to include the `mdc-card__action` class on every action for correct positioning. Also, be sure to include the
 `mdc-button--compact` class on buttons for correct alignment.
 
+You can use the `mdc-card__actions--border` option to place a divider line to separate the `mdc-card__actions` from 
+the previous element (ex.: a `mdc-card__supporting-text`).  This should not be used with `mdc-card__actions--vertical`.
```

Comment:
Instead of saying it shouldn't be used with vertical mode, why not use the `:not()` operator defining what it shouldn't be used with in order to outright prevent browsers from using the styling where it shouldn't be. i.e. `&--border:not(&--vertical)`